.env

```diff
@@ -0,0 +1,12 @@
+BASE_IMAGE=adoptopenjdk:11-jre-hotspot
+
+XP_VERSION=7.2.1
+
+IMAGE_NAME="enonic/xp"
+IMAGE_TAG="${XP_VERSION}-ubuntu"
+IMAGE_FULL_TAG="${IMAGE_NAME}:${IMAGE_TAG}"
+
+DOCKER_BUILD_EXTRA_ARGS="--no-cache --pull"
+
+PUSH_TO="docker.io/$IMAGE_FULL_TAG"
+#PUSH_TO="docker.io/$IMAGE_FULL_TAG gcr.io/public-resources-235813/$IMAGE_FULL_TAG"
```

Comment:
Do we need this commented out line (usually repo name is a secret, although I don't know why)

Reply:
I do not think we need it but it was in the old script, so I kept it in. Maybe @diegopasten knows why it was there in the first place.